codice/ddf
# [2.19.x] DDF-6152 Added standalone Solr gazetteer

#### What does this PR do?
Backport of #6153 

This adds a new gazetteer feature that copies all gazetteer data into a separate solr core called `gazetteer`. This allows the index to be updated without the overhead of all other items in the catalog.

#### Who is reviewing it? 


#### Select relevant component teams: 


#### Ask 2 committers to review/merge the PR and tag them here.


#### How should this be tested?


- Stop the `DDF :: Spatial :: Geocoding :: Plugin` bundle
- Install the Solr Gazetteer feature `feature:install -r catalog-solr-offline-gazetteer`
- Ingest a record with the `gazetteer` metacard tag
- Verify the document is eventually created at https://localhost:8994/solr/#/gazetteer/query
- Verify the DDF UI location lookup can find the record

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: n/a

#### Screenshots


#### Checklist:
- [ n/a ] Documentation Updated
- [ n/a ] Update / Add Threat Dragon models
- [ n/a ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
